cmd/edgectl/aes_install.go

```diff
@@ -987,24 +1009,35 @@ Removing the CRDs will cause your existing Ambassador Mappings and other resourc
 
 $ kubectl delete crd -l product=aes
 
-Aborting the installer to avoid corrupting an existing (but undetected) installation.
+The installer will now quit to avoid corrupting an existing (but undetected) installation.
 `
 
-const seeDocs = "See https://www.getambassador.io/docs/latest/tutorials/getting-started/"
+const seeDocs    = "See https://www.getambassador.io/docs/latest/tutorials/getting-started/"
```

Comment:
Should seeDocs somehow reference or build on seeDocsURL (so the URL string doesn't exist in two places)?

Reply:
I was thinking about that too.  I'll see if Go will let me construct a const from string fragments, otherwise I'll have to pass it in as a parameter.

Reply:
That feature exists in the code I'm working on. I think we can get Bruce's PR into 1.3.2, whereas my work will take longer. Let's leave in the duplication for now and improve things in the next release.

Reply:
fixed already in this PR